tasks/deploy/update.js

```diff
@@ -50,5 +54,30 @@ module.exports = function (gruntOrShipit) {
         shipit.log(chalk.green('Finished copy.'));
       });
     }
+
+    function setPreviousRevision() {
+      return shipit.getPreviousReleaseDirname().then(function(previousReleaseDir) {
+        if (previousReleaseDir) {
+          var file = path.join(shipit.releasesPath, previousReleaseDir, 'REVISION');
+          return shipit.remote('if [ -f ' + file + ' ]; then cat ' + file + ' 2>/dev/null; fi;').then(function(response) {
+
+            // TODO: How should we handle multiple?
```

Comment:
`shipit.config.servers`. Should we just grab the result from the first, as I'm doing, assuming they should all be the same?